theories/crypto/PKE.ec

```diff
@@ -6,7 +6,11 @@
  * Distributed under the terms of the CeCILL-B-V1 license
  * -------------------------------------------------------------------- *)
 
-require import AllCore List Distr DBool LorR.
+require import AllCore List Distr DBool.
+
+require LorR. 
```

Comment:
`LorR` should now be abstract, since it shouldn't be used without instantiating `input`.

Reply:
Just to clarify on this one: please @mortensol rename the `LorR.ec` file as `LorR.eca` and fix things that break. (I don't think anything will break that you haven't already fixed.)

Reply:
``LorR`` is now abstract.